### PR TITLE
Design changes to content on 'Programas.html'

### PR DESCRIPTION
**Pull request by anxelm**
Hey,

I made some design changes to the content of 'programas.html'. I thought it'd be more pleasing to the eye of the visitor of the web page.

Hope you like it, this is my very first commit to an open project.

Best regards,

### CODE REVIEW

File: programas.html [hunk 1]
```diff
@@ -1,16 +1,45 @@
 ---
 layout: default
 ---
+<head>
```
Comment (wOvalle) [inline programas.html]: Add custom css into https://github.com/fundacionmamasena/fundacionmamasena.github.io/blob/master/css/custom.css, so we don't have to insert two <head> tags in the page

File: programas.html [hunk 2]
```diff
@@ -1,16 +1,45 @@
...
+	<style> 
+	div.cities {
```
Comment (wOvalle) [inline programas.html]: Why this class is called cities? Shouldn't be like "spaces" or "programs-paragraphs"?

File: programas.html (hunk 1 above)
Comment (anxelm) [inline programas.html]: Done.

File: programas.html (hunk 2 above)
Comment (anxelm) [inline programas.html]: Done.

**Pull request merged**